packages/flutter/test/material/bottom_app_bar_test.dart

```diff
@@ -218,19 +284,18 @@ void main() {
                 ),
                 bottomNavigationBar: BottomAppBar(
                   color: Color(0xff0000ff),
+                  surfaceTintColor: Colors.transparent,
```

Comment:
I add this `surfaceTintColor: Colors.transparent` in a few tests of the `BottomAppBar`'s base color, when the by-default surface tint color would make that more complicated.

<details>

<summary>The doc on `surfaceTintColor` seems wrong:</summary>

https://api.flutter.dev/flutter/material/BottomAppBar/surfaceTintColor.html
> If this is null, no overlay will be applied.

https://github.com/flutter/flutter/blob/5054b6e51/packages/flutter/lib/src/material/bottom_app_bar.dart#L193
```dart
final Color surfaceTintColor = widget.surfaceTintColor ?? babTheme.surfaceTintColor ?? defaults.surfaceTintColor!;
```

If it's null, then an overlay _will_ apply: one from the ambient `BottomAppBarTheme`, else from a hard-coded default for M2 or M3.

</details>

Reply:
Could you file a new issue about the docs being wrong?

Reply:
Sure, I'll aim to do that today.

Reply:
#124535